### ISSUE TITLE
What are the default hyperparameters for training mobilenet on imagenet?

### ISSUE THREAD

**Issue by kwotsin** [Zehaos/MobileNet#33]
I think in the mobilenet paper, it is mentioned that they trained in a similar way as the inception v3. In the inception V3 paper, they used a learning rate of learning rate decay of 0.94 every 2 epochs, which is what you defined here: https://github.com/Zehaos/MobileNet/blob/master/train_image_classifier.py

However, on the training script here: https://github.com/Zehaos/MobileNet/blob/fce86531a7a93ccee5e31893e8587b7076889f24/scripts/train_mobilenet_on_imagenet.sh

the learning rate is instead 0.1 (inception v3 used 0.045), and the decay rate is 0.1 every 30 epochs. Which one gave you a better performance? I am testing the retraining performance, and using the 0.1 learning rate and 0.1 decay every 30 epochs seem to give me a less optimal performance in the loss compared to the loss graph you shown. Also, in total, how many epochs did you train the model for?

Thanks for your help. The imagenet training is indeed not as easy with just 1 GPU.

**Comment by Zehaos** [comment]
@kwotsin I trained it for about 120 epoch in total (I stop it because I can't wait any more), got 66.51% acc without tuning any hyper-params. 

I use 0.01 as the init learning rate in fact, the script had been revised for several time (you can check out the history).

**Comment by Zehaos** [comment]
@kwotsin Sorry, I have forget the hyper-params setting. I checked out some related issues, may be 0.1 instead.

**Comment by kwotsin** [comment]
@Zehaos No worries about it. I think generally the YellowFin performance seems to be much better, but I still haven't got the opportunity to test it out past 6k steps (training takes a long time).
